### DISCUSSION TITLE
pyscript now in the myst-parser (sphinx) documentation

### DISCUSSION THREAD

**Discussion by chrisjsewell** [pyscript/pyscript#1239] [Show and tell]
Hi all 👋 ,

Especially as I see you are using mayst-parser for your documentation 😄 
I thought I would share that I created https://sphinx-pyscript.readthedocs.io, primarily in order to create a live preview within the myst-parser documentation (of MyST -> HTML) https://myst-parser.readthedocs.io/en/latest/live-preview.html ⚡

**Comment by JeffersGlass** [comment]
Wow! This is really awesome!

Does the code for that wheel live online somewhere? The GitHub link from the PyPI page seems to be broken.

**Reply by chrisjsewell** [reply]
Heya, You mean this https://github.com/sphinx-extensions2/sphinx-pyscript? (I recently moved it over from my personal account)

**Reply by chrisjsewell** [reply]
Happy if you wanna suggest any improvements, or collab on it 😄
